kube-derive/src/custom_resource.rs

```diff
@@ -239,12 +240,14 @@ pub(crate) fn derive(input: proc_macro2::TokenStream) -> proc_macro2::TokenStrea
     }
 
     let docstr = format!(" Auto-generated derived type for {ident} via `CustomResource`");
+    let quoted_serde = Literal::string(&serde.to_token_stream().to_string());
     let root_obj = quote! {
         #[doc = #docstr]
         #[automatically_derived]
         #[allow(missing_docs)]
         #[derive(#(#derive_paths),*)]
         #[serde(rename_all = "camelCase")]
+        #[serde(crate = #quoted_serde)]
```

Comment:
This makes sense to me. If it's part of the default flow and it works for that then it's kind of already tested.